### ISSUE TITLE
The Unity SDK automatically enables the JNI debugging

### ISSUE THREAD

**Issue by alonsohki** [facebook/facebook-sdk-for-unity#258]
https://github.com/facebook/facebook-sdk-for-unity/blob/0469ebf68965263e90063c58e9f4426c5a5f846b/Facebook.Unity/Mobile/Android/AndroidFacebookGameObject.cs#L30

Is there any reason why this is being forced onto the app?

**Comment by bmadsen-assf** [comment]
Indeed, please remove this.  It's beyond annoying to have this turned on.

**Comment by KylinChang** [comment]
We will remove in the next release (v7.18.1)

**Comment by jjfmarket** [comment]
Still seeing this behaviour
